llvm/test/Transforms/FunctionAttrs/sendmsg-nocallback.ll

```diff
@@ -0,0 +1,86 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-attributes --check-globals all --version 5
+; RUN: opt -S -passes=function-attrs < %s | FileCheck --check-prefixes=COMMON,FNATTRS %s
+; RUN: opt -S -passes=attributor-light < %s | FileCheck --check-prefixes=COMMON,ATTRIBUTOR %s
+
+define internal void @sendmsg_is_norecurse() {
```

Comment:
Is the point of these tests that the intrinsics are nocallback, and nocallback implies norecurse, so we can infer that the functions are norecurse?

Reply:
Yes